src/components/LoggedinHeader/LoggedinHeader.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import loggedinheaderLogo from "../../images/NewsExplorer-min.svg";
+import hamburgerMenuOpen from "../../images/menu_dark.svg";
+import hamburgerMenuClose from "../../images/close-min.svg";
+import "../LoggedinHeader/LoggedinHeader.css";
+import logoutDark from "../../images/logout_dark.svg";
+
+
+function LoggedinHeader({ name, onLogout }) {
```

Comment:
You need to **reuse** component `Header` for any header. You shouldn't duplicate the markup

Reply:
Done